test/e2e/framework/metrics/kubelet_metrics.go

```diff
@@ -36,9 +36,22 @@ func NewKubeletMetrics() KubeletMetrics {
 
 // GrabKubeletMetricsWithoutProxy retrieve metrics from the kubelet on the given node using a simple GET over http.
 // Currently only used in integration tests.
-func GrabKubeletMetricsWithoutProxy(nodeName string) (KubeletMetrics, error) {
-	metricsEndpoint := "http://%s/metrics"
-	resp, err := http.Get(fmt.Sprintf(metricsEndpoint, nodeName))
+func GrabKubeletMetricsWithoutProxy(endpoint string) (KubeletMetrics, error) {
```

Comment:
is there a reason why we don't just have `node` and `path` arguments here?  It probably looks cleaner on the calling side